grpc/send_incoming_call_notification.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"log"
```

Comment:
This was marked as resolved, but `log` is still being used when Plogger is preferred. Logs from the `log` module will not be tracked in the usual log files we manage. This appears in other files as well, only commenting it once

Reply:
Whoops, slipped my brain a lil. These are now all plogger excluding the protocol / method logging in `server.go`, as in every other server that continues to use fmt.